styleguide/index.html

```diff
@@ -43,10 +43,10 @@ <h1 class="kss-doc-title">GovCMS 8 UI-Kit Starter Style Guide</h1>
         <div id="kssref-0" class="kss-section kss-section--depth-0 kss-overview kss-style">
       <h1>GovCMS 8 UI-Kit Starter Styleguide</h1>
 
-<p>This styleguide is showing only the UI-Kit components with a reasonable 
-template. It is not providing a comprehensive list of every style 
+<p>This styleguide is showing only the UI-Kit components with a reasonable
+template. It is not providing a comprehensive list of every style
 available in UI-Kit or the theme. Feel free to add other components as needed.</p>
-<p>For live examples of other styling, install the defaul content and visit 
+<p>For live examples of other styling, install the defaul content and visit
```

Comment:
@enzolutions Can you please fix the typo in this line while you are at it - defaul -> default

cc: @fubarhouse

Reply:
done